scripts/ci/test/test-ci.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euo pipefail
+
+script_dir=$(cd -- $(dirname -- "${BASH_SOURCE[0]}") && pwd)
+
+if ! command -v gh >/dev/null 2>&1; then
+    printf "\n\t*** ${LIGHT_RED}"gh" not found. Please install github cli${NC} ***\n\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)